src/explore-education-statistics-admin/src/prototypes/components/PrototypeMetaCreate.tsx

```diff
@@ -0,0 +1,192 @@
+          with further detail on the methods used. The underlying data files
+          include national, regional, local authority, local authority district
+          and school level absence information from 2006/07 to 2018/19 for
+          schools in England.
+        </p>
+        <p></p>
+        <hr />
+        <h2>File format and conventions</h2>
+        <h3 className="govuk-heading-s">Rounding</h3>
+        <p>This dataset has not had suppression applied.</p>
+        <h3 className="govuk-heading-s">Conventions</h3>
+        <p>The following convention is used throughout the underlying data</p>
+        <hr />
+      `,
+    },
+    descriptionPlaceholder: {
+      text: `
+      <h2 class="govuk-heading-m">Description</h2>
+      <p>---</p>
+      <h2 class="govuk-heading-m">Coverage</h2>
+      <p>---</p>
+      <h2 class="govuk-heading-m">File format and conventions</h2>
+      <p>---</p>
+      `,
+    },
+    subject1: {
+      text:
+        'Absence information for all enrolments in state-funded primary, secondary and special schools including information on overall absence, persistent absence and reason for absence for pupils aged 5-15, based on all 5 half terms data from 2006/07 to 2011/12 inclusive and based on 6 half term data from 2012/13 onwards',
+    },
+    subject2: {
+      text:
+        'Absence information by pupil characteristics such as age, gender and ethnicity by local authority.',
+    },
+    subject3: {
+      text:
+        'Absence information by pupil characteristics such as age, gender and ethnicity by local authority district',
+    },
+  };
+
+  return (
+    <>
+      {createMeta && !dialog && (
+        <>
+          <div className="govuk-inset-text">
+            <h2 className="govuk-heading-m">Before you start</h2>
+            <ul className="govuk-list--bullet">
+              <li>
+                ensure you have uploaded all your subject files before creating
+                the public metadata document
+              </li>
+              <li>
+                you can create the document and change subject files later, but
+                you may need to re-edit the content
+              </li>
+            </ul>
+          </div>
+
+          <button
+            className="govuk-button"
+            type="submit"
+            onClick={() => {
+              setAddNewMeta(true);
+              setCreateMeta(false);
+              setEditMeta(false);
+            }}
+          >
+            Create public metadata
+          </button>
+        </>
+      )}
+      {addNewMeta && (
+        <>
+          <CreateMetaForms
+            editing={editMeta}
+            description={
+              editMeta
+                ? formText.descriptionSaved.text
+                : formText.descriptionPlaceholder.text
+            }
+            subject1={formText.subject1.text}
+            subject2={formText.subject2.text}
+            subject3={formText.subject3.text}
+          />
+          <button
```

Comment:
Would suggest rather than having one big form, we could have individual save/cancel buttons on each file section and the meta description. Otherwise the user has to potentially scroll too much and also could make unintended changes more easily.

Reply:
added place holder buttons for each section (they don't do anything at the mo - just show positioning)